### ISSUE TITLE
Provide loss function (FocalLoss)

### ISSUE THREAD

**Issue by hienpham15** [Praneet9/Representation-Learning-for-Information-Extraction#38]
I have noticed that in train.py and eval.py you have import a FocalLoss class which is then used as criterion for both training and evaluating.

But I couldn't find relevant definition of this loss function in both the paper and the source code.

Would you mind sharing the FocalLoss class definition ?

**Comment by Praneet9** [comment]
@hienpham15 The paper doesn't use FocalLoss, it was something we did as an addition to the model training due to the skewed dataset we trained on. It was easier to use a prebuilt library for focal loss instead of writing our own. You can feel free to use any loss function for your convenience.
